airdnb/src/main/java/team10/airdnb/reservation/entity/Reservation.java

```diff
@@ -43,11 +45,28 @@ public class Reservation {
     private LocalDate checkOutDate;
 
     @Column(name = "capacity")
-    private long capacity;
+    private Integer capacity;
 
     @Column(name = "is_confirmed")
     private boolean isConfirmed;
 
     @Column(name = "total_price")
-    private long totalPrice;
+    private BigDecimal totalPrice;
+
+    // 숙박 총 비용을 계산
+    public static BigDecimal calculateTotalPrice(Accommodation accommodation, LocalDate checkInDate, LocalDate checkOutDate) {
```

Comment:
```suggestion
    public static BigDecimal calculateTotalPrice(AccommodationFee accommodationFee, LocalDate checkInDate, LocalDate checkOutDate) {
```

차라리 인자로 AccommodationFee 를 받는건 어떨까요?